### ISSUE TITLE
Infinite Near Crit

### ISSUE THREAD

**Issue by Swagile** [HippieStationCode/HippieStation13#3076]
If you are in near crit, you cannot succumb and are forced to ghost if you want to "die". This means people can grief you by keeping you in a constant state of near crit with medication. Happened to me ages ago by a medical doctor that pumped me with epherdine but then died to a traitor, leaving me sitting there with no ability to do anything for a long ass time until I could finally succumb.

**Comment by Spacedong** [comment]
just crawl to inflict brute damage on yourself

**Comment by Spacedong** [comment]
or constantly scream to go unconscious then succumb that way

**Comment by Spacedong** [comment]
either way its not a bug

**Comment by Spacedong** [comment]
@JohnGinnane

**Comment by JohnGinnane** [comment]
Eh, the ephedrine stops you taking further damage I believe

**Comment by Swagile** [comment]
yeah, that and salbutomol or whatever fucking name the oxygen healing chem is

keeps you in crit 5ever so unless you suicide or ghost (giving up the chance at being revived) then ur fucked

screaming doesn't help when the oxygen shit heals you constantly and epherdine does the same

**Comment by ArcLumin** [comment]
So either suicide typing whisper while in crit, which doesn't make you unavailable for cloning, wait for someone to actually heal you, or crawling outside an airlock, or over glass
